milk/package.json

```diff
@@ -16,8 +16,11 @@
     "typescript": "^4.9.4"
   },
   "scripts": {
-    "build": "rm -rf dist/ && tsc",
-    "serve": "node --enable-source-maps dist/index.js"
+    "build": "npm run clean:win && npm run clean:lin && tsc",
+    "serve": "node --enable-source-maps dist/index.js",
+    "clean": "npm run clean:win && npm run clean:lin",
+    "clean:win": "node -e \"if (process.platform === 'win32') process.exit(1)\" || , if exist dist rmdir /Q /S dist",
+    "clean:lin": "node -e \"if (process.platform !== 'win32') process.exit(1)\" || rm -rf dist"
```

Comment:
Might be worth using a devDependency like [rimraf](https://github.com/isaacs/rimraf) to avoid blowing up the list with OS-specific commands? I see it used often used for such use cases, called like `rimraf dist/`.